common/fees.js

```diff
@@ -3,6 +3,7 @@ import network from '~/network'
 export default {
   ...network.fees,
   getFees(transactionType) {
+    // Use default fees if transactionType is not 'ClaimRewardsTx'
```

Comment:
Actually, I find this comment misleading, as the fee is obtained in `network.fees`